doc/languages-frameworks/cuda.section.md

```diff
@@ -54,3 +54,52 @@ for your specific card(s).
 
 Library maintainers should consult [NVCC Docs](https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/)
 and release notes for their software package.
+
+## Adding a new CUDA release (#adding-a-new-cuda-release)
+
+> **WARNING**
+>
+> This section of the docs is still very much in progress. Apologies.
+
+For the CUDA Toolkit:
```

Comment:
I'd move this subsection to after redist and rename it into something literally repulsive, like "to update the deprecated runfile-based cudatoolkit installation"

Reply:
d990c03ed934056291dc3d79d00eb52a2189968d